enterprise/helpers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for enterprise app.
+"""
+
+from .models import EnterpriseCustomerBrandingConfiguration
```

Comment:
Why? (Relative imports are IMHO easier to read and require fewer changes when packages are renamed/moved. I believe the edX style guide allows either absolute or relative imports.)